app/javascript/ui/casebooks/dragging.coffee

```diff
@@ -0,0 +1,97 @@
+ui = require 'ui/turbolinks'
```

Comment:
I defaulted to coffeescript since it's the default in new Rails projects. (At least I think that's still true?) 

I like CS for some of its utility features that replace a big library like Underscore, but I'm not wedded to it if you would be more comfortable working in pure JS or ES6 with Babel.